finstmt/findata/database.py

```diff
@@ -134,3 +127,39 @@ def get_sympy_subs_dict(self, t_offset: int = 0) -> Dict[IndexedBase, float]:
                 self.prior_statement.get_sympy_subs_dict(t_offset=t_offset - 1)
             )
         return subs_dict
+
+    #### NEW
```

Comment:
No need for comments like this, unless it was just for your dev purpose.

Reply:
agree. removed.
https://github.com/nickderobertis/py-finstmt/pull/100/commits/16e7f34f6884cbb438f87f79b4da290d6b2a6395